04-radio4000.sql

```diff
@@ -82,7 +82,8 @@ create table tracks (
 	url text not null,
 	discogs_url text,
 	title text not null,
-	description text
+	description text,
+	tags text[]  default '{}'
```

Comment:
not sure if we want this default value? why? Found it on the internet

Reply:
tbh, no idea what it means in term of sql